pkgs/by-name/ei/eigenlayer-cli/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
```

Comment:
```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:

```

Reply:
Is that standard? Which formatter do that? Fixed

Reply:
I use `nixpkgs-fmt`, but the main idea is the extra space after the `}:` and the `buildGoModule`

Reply:
Oh, got that, I will try to use `nixpkgs-fmt` for next PRs.